ISHLogDNAService/ISHLogDNAService.m

```diff
@@ -214,7 +214,7 @@ + (void)setEnabled:(BOOL)enabled {
 }
 
 - (NSURL *)baseUrl {
-    NSString *url = [NSString stringWithFormat:@"https://logs.logdna.com/logs/ingest?hostname=%@&now=%@", self.hostName, @([[NSDate date] timeIntervalSince1970])];
+    NSString *url = [NSString stringWithFormat:@"https://logs.logdna.com/logs/ingest?hostname=%@&now=%@", [self.hostName stringByAddingPercentEncodingWithAllowedCharacters:[NSCharacterSet URLQueryAllowedCharacterSet]], @([[NSDate date] timeIntervalSince1970])];
```

Comment:
While this will most-likely work, I would recommend to adopt the URL Query Item API (in conjunction with URL Components) to leave the encoding details to the platform :)

See: https://developer.apple.com/documentation/foundation/urlqueryitem

(this is something we should have done all along, but I think it's a good time to change it, if you don't mind)